src/epitools/_sample_data.py

```diff
@@ -10,11 +10,72 @@
 
 from pathlib import Path
 
-from skimage.io import imread
+import numpy as np
+import pandas as pd
+
+import epitools._reader
 
 
 def load_sample_data():
-    """Generates an image"""
+    """Load a sample dataset"""
+
     img_path = Path("sample_data") / "8bitDataset" / "test_image.tif"
-    data = imread(img_path)
-    return [(data, {"name": "Epitools test data"})]
+    return epitools._reader.reader_function(path=img_path.as_posix())
+
+
+def load_projected_data():
+    """Load a smaple projected dataset"""
+
+    img_path = Path("sample_data") / "8bitDataset" / "test_image-projected.tif"
+    return epitools._reader.reader_function(path=img_path.as_posix())
+
+
+def load_segmented_data():
+    """Load a sample segmented dataset"""
+
+    # Load labels
+    labels_path = (
+        Path("sample_data") / "8bitDataset" / "test_image-projected-segmented.tif"
+    )
+    labels_data, labels_kwargs, labels_layer_type = epitools._reader.reader_function(
+        path=labels_path.as_posix(),
+    )[0]
+
+    labels_kwargs["name"] = "Cells"
+    labels_kwargs["metadata"]["cell_statistics"] = _load_cell_statistics()
+
+    # Load seeds
+    seeds_path = (
+        Path("sample_data") / "8bitDataset" / "test_image-projected-segmented-seeds.npy"
+    )
+    seeds_data = np.load(seeds_path)
+    seeds_kwargs = {"name": "Seeds", "translate": labels_kwargs["translate"]}
+    seeds_layer_type = "points"
+
+    seeds_kwargs["name"] = "Seeds"
+    seeds_kwargs["translate"] = labels_kwargs["translate"]
+
+    # arrange data in format required by napari
+    labels_layer_data = (labels_data, labels_kwargs, labels_layer_type)
+    seeds_layer_data = (seeds_data, seeds_kwargs, seeds_layer_type)
+
+    return [seeds_layer_data, labels_layer_data]
```

Comment:
Any particular reason for returning as a `list` rather than, say, a `tuple`?

Reply:
napari expects the returned data to be in a list - I haven't checked if a tuple works, but napari is picky sometimes so I just went with a list